Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM ubuntu:18.04
 ARG RUST_TOOLCHAIN="1.46"
```

Comment:
If we are setting this as default in ./docker.sh, should we make it required here?

Reply:
So do you want to remove it from here? That makes sense.